lib/dal/tests/integration_test/frame.rs

```diff
@@ -556,14 +581,22 @@ async fn output_sockets_can_have_both(ctx: &mut DalContext) {
 
 #[test]
 async fn simple_down_frames_no_nesting(ctx: &mut DalContext) {
-    let even_frame = create_component_for_schema_name(ctx, "large even lego", "even").await;
+    let even_frame = create_component_for_schema_name(ctx, "large even lego", "even")
+        .await
+        .expect("could not create component");
     let even_frame_component_id = even_frame.id();
-    let _ = even_frame
+    even_frame
         .set_type(ctx, ComponentType::ConfigurationFrameDown)
-        .await;
+        .await
+        .expect("could not set type");
```

Comment:
Now, we no longer swallow errors when setting type in frame tests. This is one example.